Xamarin.Forms.Platform.Android/Renderers/ProgressBarRenderer.cs

```diff
@@ -45,6 +50,38 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 
 			if (e.PropertyName == ProgressBar.ProgressProperty.PropertyName)
 				UpdateProgress();
+			else if (e.PropertyName == ProgressBar.ProgressColorProperty.PropertyName)
+				UpdateProgressColor();
+		}
+
+		void UpdateProgressColor()
+		{
+			if (Element == null || Control == null)
```

Comment:
Why do you need to check these here? Shouldn't `Element` and `Control` be initialized whenever this method is called?

Reply:
I thought about keeping it consistent with the **Android ActivityIndicatorRenderer** `UpdateColor` method (line 54). Anyhow, at this moment there are no code paths in both Renderers that take to a null `Element` and/or `Control`.